### PR TITLE
Use BEARER_TOKEN_FILE env var to specify the token path

### PR DESCRIPTION
**Pull request by brianhlin**
Token discovery details: https://zenodo.org/record/3937438. Support was added for this to `condor_submit` in 9.0.2 and is expected in 9.1.3. I'm working on making use of this in `condor_ce_run` and adding token discovery to `condor_ce_trace`

### CODE REVIEW

File: osgtest/tests/test_410_jobs.py [hunk 1]
```diff
@@ -12,6 +12,16 @@
 
 class TestRunJobs(osgunittest.OSGTestCase):
 
+    def tearDown(self):
+        env_vars = ('_condor_SCITOKENS_FILE',
+                    'BEARER_TOKEN_FILE')
+
+        for env_var in env_vars:
+            try:
+                os.environ.pop(env_var)
```
Comment (matyasselmeci) [inline osgtest/tests/test_410_jobs.py:21]:
```suggestion
                del os.environ[env_var]
```
pop() is only if you also want the value back.
Alternatively you could use
```
os.environ.pop(env_var, None)
```
in which case you can drop the KeyError check

Comment (brianhlin) [reply osgtest/tests/test_410_jobs.py]: Fixed! The fewer tabs the better

File: osgtest/tests/test_410_jobs.py [hunk 2]
```diff
@@ -12,6 +12,13 @@
 
 class TestRunJobs(osgunittest.OSGTestCase):
 
+    def tearDown(self):
+        env_vars = ('_condor_SCITOKENS_FILE',
+                    'BEARER_TOKEN_FILE')
+
+        for env_var in env_vars:
+            os.environ.pop(env_var, None)
```
Comment (edquist) [inline osgtest/tests/test_410_jobs.py:20]: I was going to make the same comment about `del d[k]`.  Looks great how you've done it though.

> The fewer tabs the better

well if that's the case...

```suggestion
        [ os.environ.pop(env_var, None) for env_var in env_vars ]
```

(whose returned list will also be thrown to the wind)

**Review by edquist — APPROVED** [review]
lgtm

**Pull request merged**